### PR TITLE
Fixes: #17023 Target Dimension not set

### LINKED ISSUE #17023: Target dimension not set
### Describe the bug

I was on Monitor page, while doing a lot of queries continuously (99.9% of them `INSERT`) when I received this error. Is the first time I got it. So I don't know how to reproduce it. Questions chart is the only one that didn't work.

### To Reproduce

1. Go to `Monitor` page
2. Run millions of `INSERT` queries
3. Wait for the error, if it happens

### Expected behavior

Questions chart should be working and that JavaScript error shouldn't pop.

### Screenshots

s1

### Server configuration

- Operating system: CentOS 8
- Web server: nginx/1.14.1
- Database version: 5.7.35
- PHP version: 8.0.8
- phpMyAdmin version: 5.2.0-dev+20210717.41c478edd7

### Client configuration

- Browser: Firefox 92
- Operating system: Windows 10

### Additional context

```json
{
    "pma_version": "5.2.0-dev+20210717.41c478edd7",
    "browser_name": "FIREFOX",
    "browser_version": "92.0",
    "user_os": "Win",
    "server_software": "nginx/1.14.1",
    "user_agent_string": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0",
    "locale": "en",
    "configuration_storage": "enabled",
    "php_version": "8.0.8",
    "script_name": "index.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "Error",
        "message": "Target dimension not set",
        "stack": [
            {
                "func": "jqPlot/this.quickInit",
                "line": "2387",
                "column": "23",
                "context": [
                    "            ",
                    "            this._height = this.target.height();",
                    "            this._width = this.target.width();",
                    "            ",
                    "            if (this._height <=0 || this._width <=0 || !this._height || !th//...",
                    "                throw new Error(\"Target dimension not set\");",
                    "            }",
                    "            ",
                    "            this._plotDimensions.height = this._height;",
                    "            this._plotDimensions.width = this._width;",
                    "            this.grid._plotDimensions = this._plotDimensions;"
                ],
                "uri": "js/vendor/jqplot/jquery.jqplot.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jqplot/jquery.jqplot.js"
            },
            {
                "func": "jqPlot/this.replot",
                "line": "2888",
                "column": "22",
                "context": [
                    "            // otherwise, quickinit.",
                    "            if (data || !$.isEmptyObject(opts)) {",
                    "                this.reInitialize(data, opts);",
                    "            }",
                    "            else {",
                    "                this.quickInit();",
                    "            }",
                    "",
                    "            if (resetAxes) {",
                    "                this.resetAxesScale(resetAxes, opts.axes);",
                    "            }"
                ],
                "uri": "js/vendor/jqplot/jquery.jqplot.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jqplot/jquery.jqplot.js"
            },
            {
                "func": "refreshChartGrid/runtime.refreshRequest</<",
                "line": "1645",
                "column": "22",
                "context": [
                    "        }",
                    "",
                    "        i++;",
                    "",
                    "        if (runtime.redrawCharts) {",
                    "          elem.chart.replot();",
                    "        }",
                    "      });",
                    "      oldChartData = chartData;",
                    "      runtime.refreshTimeout = setTimeout(refreshChartGrid, monitorSettings//...",
                    "    });"
                ],
                "uri": "js/dist/server/status/monitor.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/dist/server/status/monitor.js"
            },
            {
                "func": "each",
                "line": "2",
                "column": "3053",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "refreshChartGrid/runtime.refreshRequest<",
                "line": "1555",
                "column": "9",
                "context": [
                    "      var i = 0;",
                    "      var diff;",
                    "      var total;",
                    "      /* Update values in each graph */",
                    "",
                    "      $.each(runtime.charts, function (orderKey, elem) {",
                    "        var key = elem.chartID; // If newly added chart, we have no data fo//...",
                    "",
                    "        if (!chartData[key]) {",
                    "          return;",
                    "        } // Draw all series"
                ],
                "uri": "js/dist/server/status/monitor.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/dist/server/status/monitor.js"
            },
            {
                "func": "c",
                "line": "2",
                "column": "28327",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "fireWith",
                "line": "2",
                "column": "29072",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "l",
                "line": "2",
                "column": "79901",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "o/<",
                "line": "2",
                "column": "82355",
                "context": [
                    "/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=5.2.0-dev%2B20210717.41c478edd7",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            }
        ],
        "uri": "index.php?route=%2Fserver%2Fstatus%2Fmonitor"
    }
}
```

### PR DESCRIPTION
**Pull request by shucontech**
Signed-off-by: Saksham Gupta <shucontech@gmail.com>

### Description

Please describe your pull request.

Fixes #17023

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: themes/pmahomme/scss/_common.scss [hunk 1]
```diff
@@ -818,6 +818,7 @@ table#chartGrid {
   div.monitorChart {
     background: #ebebeb;
     border: none;
+    min-width: 1px;
```
Comment (williamdes) [inline themes/pmahomme/scss/_common.scss:821]: No other themes affected?
Is QA_5_1 not affected?

Comment (shucontech) [reply themes/pmahomme/scss/_common.scss:821]: Ahh, I forgot about the other themes. I'll check it once. Should I rebase to QA_5_1?

**Comment by shucontech** [comment]
Closing in favour of https://github.com/phpmyadmin/phpmyadmin/pull/17102

**Comment by shucontech** [comment]
I couldn't replicated the issue in Metro theme, but I've added the fix anyways.

**Pull request merged**